src/types.ts

```diff
@@ -61,19 +59,9 @@ export interface FiatConnectApiClient {
   ): Promise<Result<TransferStatusResponse, ResponseError>>
 }
 
-// These must be manually updated as more KYC and FiatAccount types become standardized
-export type KycSchemaData = PersonalDataAndDocumentsKyc // in the future this will be the union of all KYC schema types (currently there is just one)
-export type FiatAccountSchemaData = // similarly, this will be the union of all fiat account schema types
-  AccountNumber | MobileMoney | DuniaWallet | IBANNumber | IFSCAccount
-
-export interface AddKycParams {
-  kycSchemaName: KycSchema
-  data: KycSchemaData
-}
-
-export interface AddFiatAccountParams {
-  fiatAccountSchemaName: FiatAccountSchema
-  data: FiatAccountSchemaData
+export interface AddKycParams<T extends KycSchema> {
```

Comment:
is there a reason this isn't in types as well? looks pretty similar to PostFiatAccountRequestBody, wondering if it should be located in the same place

Reply:
not a big deal though

Reply:
This is because POST /kyc still accepts schema name via url param unlike POST /accounts which takes schema name and data in the body. This is just an sdk helper type to pass kyc schema name and body in a single object.